src/main/java/core/basesyntax/RandomGenerator.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RandomGenerator {
+    private static final int TOTALNUMBER = 5;
```

Comment:
```suggestion
    private static final int TOTAL_NUMBER = 5;
```
let's improve naming, it's not clear total number of what?